File: easy/distribution_of_sweets/distribution_of_sweets.php
<!DOCTYPE html>
<html>
<head>
	<title>Distribution of sweets</title>
</head>
<body>
	<form  method="post">
		<label for="siblings">Number of siblings: </label>
		<input type="text"  name="siblings">

		<label for="popsicles">Number of popsicles: </label>
		<input type="text"  name="popsicles">

		<input type="submit" value="Action" >
	</form>	
	<br>
	<?php

		function distribution($siblings,$popsicles){
			if ($siblings == NULL || $popsicles == NUll ){
				return("");
			}

			if ($popsicles%$siblings == 0){
				echo "give away";
			}else{
				echo "eat them yourself";
			}

	    }

	    $siblings = $_POST["siblings"];
	    $popsicles = $_POST["popsicles"];

	    distribution($siblings,$popsicles);
        
	?>

</body>
</html>
		

File: easy/distribution_of_sweets/distribution_of_sweets.c
#include <stdio.h>
#include <math.h>

void main() {

    int siblings,popsicles;
    printf("Introduce el número de hermanos y de dulces: ");
    scanf("%d%d",&siblings,&popsicles);

    if(popsicles%siblings == 0) printf("give away");
    else printf("eat them yourself");
}
